### DISCUSSION TITLE
Audio Levels via AUDS=1 not working

### DISCUSSION THREAD

**Discussion by SndChsr** [gopro/labs#661] [Q&A] [answered]
Hello folks. I've generated the QR code for AUDS=1 for my 12 black, and using the audio connector, I'm not seeing audio levels on my screen. Isn't that what this command is supposed to do? What am I doing wrong? Can anyone help?
Many thanks!

**Comment by dnewman-gpsw** [comment]
You may have to turn on voice detection and the audio is not running without it.  Works here.

Try 
![image](https://github.com/gopro/labs/assets/10856084/142a2213-ec53-41cf-9f55-96fe0b96dcd4)

**Reply by dnewman-gpsw** [reply]
I did see bug with using AUDS without EXPS, I will see to fixing that.

**Comment by SndChsr** [comment]
Thank you so much! I think I was under the wrong impression. I thought that it was possible to enable an audio meter like one would see when using the media mod with green bars reacting to sound levels. Would something like that be possible? 
See attached picture please.

![image](https://github.com/gopro/labs/assets/152935201/2f7a3849-b873-4d53-876e-4c2c3c432050)

**Reply by dnewman-gpsw** [reply, marked as answer]
It is technically possible, but not implemented.

**Reply by SndChsr** [reply]
Thanks! This would be a great feature to have especially when using external mics.

**Reply by dnewman-gpsw** [reply]
If you using external Mics, then you have MediaMod, and then you will automatically get UV meters (as you pictured.)

**Reply by SndChsr** [reply]
I'm using the Audio Connector rather than the media mod because it has a design flaw where the gopro needs to be taken out of the media mod case in order to access the battery. When mounted to a gimbal, this would be a very tedious and time consuming task. I've petitioned Gopro to make audio levels a possibility in future firmware updates when using the audio connector.

**Reply by dnewman-gpsw** [reply]
Or do this to Media Mod to simplify camera mounting and battery exchanges.  
![image](https://github.com/gopro/labs/assets/10856084/65f27d77-f0e6-4793-80c3-91be2742d2bf)

**Reply by dnewman-gpsw** [reply]
works great.

**Reply by SndChsr** [reply]
LOL! This'll work! :)
